setup.py

```diff
@@ -15,17 +15,18 @@
     zip_safe=False,
     include_package_data=True,
     install_requires=[
-        "docopt",
-        "bs4",
-        "lxml",
-        "webvtt-py",
-        "youtube-dl",
-        "python-slugify",
-        "jinja2",
-        "mistune",
-        "requests",
-        "iso-639",
-        "zimscraperlib",
+        "docopt==0.6.2",
+        "beautifulsoup4==4.9.1",
+        "lxml==4.5.1",
+        "webvtt-py==0.4.5",
+        "python-slugify==4.0.0",
+        "Jinja2==2.11.2",
+        "mistune==0.8.4",
+        "requests>=2.24,<3.0",
+        "iso-639==0.4.5",
+        "zimscraperlib>=1.1.2,<1.2",
+        "python-magic==0.4.18",
+        "youtube_dl",
```

Comment:
Do we still need youtube_dl then?

Reply:
Yup. We use it to download some videos in some courses as they are hosted on youtube

Reply:
See https://github.com/openzim/openedx/blob/155a79b3b134649209db023ebe2d20f6d0e8a1a7/openedxtozim/xblocks_extractor/Video.py#L108

and 

https://github.com/openzim/openedx/blob/155a79b3b134649209db023ebe2d20f6d0e8a1a7/openedxtozim/utils.py#L215